src/posts/2020-04-17-announcing-subvisual-live-talks/index.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 148
+path: /posts/148-announcing-subvisual-live-talks/
+title: "Announcing Subvisual Live Talks: Zamith, Josh Clayton, Sasa Juric and others"
+author: fernando-mendes
+date: 2020-04-17
+intro: >
+  A while ago I felt the shame of copying and pasting code around from projects.
+  Eventually, I decided to stop being lazy, extracting it and making a hex
+  package out of it. This is story of that project and a walkthrough all the
+  awesome things it does, with very little code. Yay Elixir!
```

Comment:
This is the intro from your previous post about Dictator.

Reply:
I f'ed up the copy/paste. Shame. Thanks!